File: freshmen psets/2526/chapter1/examples/mario.c
// Mario by Evelyn Chan
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    int spaces = height;
    for (int row = 1; row <= height; row++)
    {
        for (int j = 0; j < spaces - row; j++)
        {
            printf(" ");
        }
        for (int hashes = 1; hashes <= row; hashes++)
        {
            printf("#");
        }
        printf("  ");
        for (int hashes = 1; hashes <= row; hashes++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: freshmen psets/2526/chapter5/sentimental/mario.py
# Hello Mario by Evelyn Chan
from cs50 import get_int

# infinite loop instead of a do while loop (which doesnt exist in python)
while True:
    height = get_int("Height: ")
    if height >= 1 and height < 9:
        break

spaces = height
for row in range(1, height + 1):
    print(" " * (spaces - row), end="")
    print("#" * row, end="")
    print("  ", end="")
    print("#" * row)
